bundles/org.openhab.ui/web/src/components/widgets/plan/oh-plan-marker.vue

```diff
@@ -136,7 +136,13 @@ export default {
           type: 'TEXT',
           context: 'color',
           description: 'Color of the tooltip'
-        }
+        },
+        {
+          name: 'zoomVisibility',
+          label: 'Zoom Visibility',
+          type: 'NUMBER',
+          description: 'Visible only when zoomed to at least this level (empty by default)'
+        },
```

Comment:
Please remove the trailing comma here, and add `advanced: true,` above (3rd line or so) as discussed above.